src/screens/login/LoginScreen.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import {Button, Container, TextInput} from '../../components'
+import { color } from '../../theme'
+import {styles } from './loginScreen.styles'
+
+export const LoginScreen = () => {
+  return (
+  <Container background={color.midGray} containerStyle={styles.container}>
+    <TextInput 
```

Comment:
<View>
 <TextInput 
        placeholder={'Email'}
        style={styles.textInput}/>
<Text>error in advance</Text>
</View>